src/styles/components/_modal.scss

```diff
@@ -137,9 +137,9 @@ $modal-padding: 25px;
 .spriteContainer {
   display: flex;
   margin: 25px;
-  align-content: center;
   align-items: center;
-  justify-content: space-between;
+  justify-content: safe center;
```

Comment:
Interesting. Never seen `safe`

Reply:
Very interesting!  https://developer.mozilla.org/en-US/docs/Web/CSS/justify-content

Reply:
Yep, the only css solution to fix our problem, where the eevee sprites were longer than the flex container!